rust/bridge/node/futures/src/executor.rs

```diff
@@ -43,29 +39,44 @@ impl<T: Future> Future for AssertSendSafe<T> {
     }
 }
 
-/// Adds support for executing closures and futures on the JavaScript main thread's microtask queue.
+/// Adds support for executing closures and futures on the JavaScript main thread's event queue.
 pub trait ContextEx<'a>: Context<'a> {
-    /// Schedules `f` to run on the microtask queue.
+    /// Schedules `f` to run on the JavaScript thread's event queue.
     ///
     /// Equivalent to `cx.queue().send(f)` except that `f` doesn't need to be `Send`.
-    fn run_on_queue(&mut self, f: impl FnOnce(TaskContext<'_>) -> NeonResult<()> + 'static) {
+    fn queue_task(&mut self, f: impl FnOnce(TaskContext<'_>) -> NeonResult<()> + 'static) {
         // Because we're currently in a JavaScript context,
         // and `f` will run on the event queue associated with the current context,
         // we can assert that it's safe to Send `f` to the queue.
         let f = AssertSendSafe(f);
         self.queue().send(move |cx| f.0(cx));
     }
 
-    /// Schedules `f` to run on the microtask queue.
+    /// Schedules `f` to run on the JavaScript thread's event queue.
     ///
     /// Equivalent to `cx.queue().send_future(f)` except that `f` doesn't need to be `Send`.
-    fn run_future_on_queue(&mut self, f: impl Future<Output = ()> + 'static) {
+    fn queue_future(&mut self, f: impl Future<Output = ()> + 'static) {
```

Comment:
No one calls `queue_task` and `queue_future`, right?

Reply:
Right; they're there for "completeness" but I could just drop them.